share/picongpu/tests/Synchrotron_plugin/include/picongpu/param/synchrotron.param

```diff
@@ -0,0 +1,78 @@
+/* Copyright 2014-2024 Filip Optolowicz
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file
+ * This file contains parameters for synchrotron extension
+ */
+
+#pragma once
+
+namespace picongpu
+{
+    namespace particles
+    {
+        namespace synchrotron
+        {
+            ///@todo remove
+            constexpr bool T_Debug = false;
+            // Carefull -> prints out a lot of stuff. Bigger simulations will produce massive outputs
```

Comment:
if you updated the main param files move the changes into this file too.

Reply:
now the file only contains the necessary params for the .ci test

Reply:
heh... can't do that.
Now it's a copy of original .param with one change